core/src/main/java/org/testcontainers/DockerClientFactory.java

```diff
@@ -95,7 +102,7 @@ public DockerClient client() {
         log.info("Docker host IP address is {}", hostIpAddress);
         DockerClient client = strategy.getClient();
 
-        if (!preconditionsChecked) {
+        if (!initialized) {
```

Comment:
Let's not do it in this PR.

Reply:
No problem, but why not? :wink:

Reply:
because it's out of scope, and if somebody submits another PR where they change DockerClientFactory, but we didn't merge "Death Note" PR yet, it will create more conflicts.
I'm happy to extract method as a follow up, or before, or in parallel, but don't want to increase the scope of this change :)

Reply:
Also think about reverts in case we discover some major bug :)

Reply:
Yes okay, makes sense :slightly_smiling_face: